bot/thiccBot/cogs/player.py

```diff
@@ -0,0 +1,130 @@
+import discord
```

Comment:
Might be a good idea to move this to the utils folder. You can do it in this pr or a follow up lol

Reply:
Yeah I will do it in a follow up. I want to add the dowloads clean up as well